src/common/HelpTrigger.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import { OverlayTrigger, OverlayTriggerProps } from 'react-bootstrap'
 
-function HelpTrigger(
-  props: Omit<OverlayTriggerProps, 'children'>
-): JSX.Element {
+type Props = {
+  className?: string
+} & Omit<OverlayTriggerProps, 'children'>
+
+function HelpTrigger({ className = '', ...props }: Props): JSX.Element {
   return (
     <OverlayTrigger placement="right" {...props}>
-      <i className="fas fa-info-circle" />
+      <i className={`${className ?? className} fas fa-info-circle`} />
```

Comment:
`?? className` is redundant. Did you mean to use something as default?

You can use [classnames](https://jedwatson.github.io/classnames/) to build the string.

Reply:
Not as default but to have the option to pass classNames to the component as props..